tests/metrics/test_stratification.py

```diff
@@ -1,371 +1,252 @@
+import itertools
 from datetime import datetime
 from math import floor
 from typing import Callable, List, Set
 from unittest.mock import MagicMock, patch
 
+import numpy as np
 import pandas as pd
 import pytest
+from vivarium import InteractiveContext
+from vivarium.framework.engine import Builder
+from vivarium.framework.population import SimulantData
+from vivarium.testing_utilities import TestPopulation, metadata
```

Comment:
Vivarium and vivarium public health both have a set of test utilities.  Vivarium provides a base population component and some tools for manipulation configuration metadata.  Vivarium public health has a mock artifact manager that supplies test data (see `vivarium_public_health.testing.mock_artifact` for details).